### ISSUE TITLE
CLI exit code

### ISSUE THREAD

**Issue by MisterMashu** [yargalot/AccessSniff#34]
Ideally, the CLI would return with an exit code of 0 if there are no errors, and 1 if there was at least 1 error.  That would make it easy to integrate it with other scripts, and especially into a continuous integration pipeline.

**Comment by VolleMelk** [comment]
Yes please! I'm having a hard time integrating this in gulp and having it fail in CI
